* Relax upper bound for py-uv-build

* Upstream uses dynamic versioning, so patch it to the actual DISTVERSION.

--- pyproject.toml.orig	2026-08-01 13:58:00 UTC
+++ pyproject.toml
@@ -1,10 +1,10 @@
 [build-system]
-requires = ["uv_build >= 0.11.32, <0.12.0"]
+requires = ["uv_build >= 0.11.32"]
 build-backend = "uv_build"
 
 [project]
 name = "napalm"
-version = "0.0.0.dev0"
+version = "%%DISTVERSION%%"
 description = "Network Automation and Programmability Abstraction Layer with Multivendor support"
 readme = "README.md"
 requires-python = ">=3.10"
